pkg/systemd/services/services.scss

```diff
@@ -4,6 +4,21 @@
 @import "./timer.scss";
 @import "../../lib/table.css";
 
+// Wrap the filter info
+.services-toolbar {
+    display: flex;
+    flex-wrap: wrap;
+    border-bottom: 1px solid var(--pf-global--BorderColor--100);
```

Comment:
Oh the border showed up in this commit too. Whoops! I'll probably be squashed down anyway... :grin:

Reply:
Sure, please remember to add a comment